peps/pep-0746.rst

```diff
@@ -0,0 +1,166 @@
+PEP: 746
+Title: TypedMetadata for type checking of PEP 593 Annotated
+Author: Adrian Garcia Badaracco <adrian@adriangb.com>
+Sponsor: Jelle Zijlstra <jelle.zijlstra@gmail.com>
+Discussions-To: https://discuss.python.org/t/XYZ
```

Comment:
Let's leave this out for now to keep the linter happy. When we merge the PR, create the discussion, and create a small PR to update the link.

```suggestion
```

Reply:
```suggestion
Discussions-To: https://discuss.python.org/t/pep-746-typedmetadata-for-type-checking-of-pep-593-annotated
```

Reply:
Thank you for all of your help @hugovk!